Assets/MixedRealityToolkit.SDK/Experimental/Features/Dwell/DwellProfile.cs

```diff
@@ -0,0 +1,91 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Input;
+using System;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Experimental.Dwell
+{
+    [MixedRealityServiceProfile(typeof(DwellProfile))]
+    [CreateAssetMenu(menuName = "Mixed Reality Toolkit/Profiles/Dwell Profile", fileName = "DwellProfile", order = 100)]
+    [Serializable]
+    public class DwellProfile : ScriptableObject
+    {
+        [Tooltip("Pointer type to use for triggering a dwell interaction")]
+        [SerializeField]
+        private InputSourceType dwellTriggerPointerType = InputSourceType.Head;
```

Comment:
>dwellTriggerPointerType [](start = 32, length = 23)

Do you think users of this feature may ever want to allow dwells with more than one input source type?  If so, then consider making this an array.

Reply:
No. This is meant to support only 1 type of input source,

---
In reply to: [313498270](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/5594#discussion_r313498270) [](ancestors = 313498270)